### PR TITLE
Feature/deploy to pages action

### PR DESCRIPTION
**Pull request by mseckykoebel**
Think I fixed the problem - dash in front of users.

### CODE REVIEW

File: .github/workflows/deploy-to-pages.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+name: Deploy to pages
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [master]
+  pull_request:
+    branches: [feature/deploy-to-pages-action]
```
Comment (jdjkelly) [inline .github/workflows/deploy-to-pages.yml]: So this needs to be changed to master now that it's a real PR

File: deploy-to-pages.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# This is a basic workflow to help you get started with Actions
```
Comment (jdjkelly) [inline deploy-to-pages.yml]: Think you accidentally committed this file?

File: .github/workflows/deploy-to-pages.yml (hunk 1 above)
Comment (mseckykoebel) [reply .github/workflows/deploy-to-pages.yml]: Ok great I think I got the flow down:

- change pull_request to whatever PR it is and open a PR on it to get it to run
- if fails fix, if not change pull_request to master and make another PR

Going to change and open a new one now

File: deploy-to-pages.yml (hunk 2 above)
Comment (mseckykoebel) [reply deploy-to-pages.yml]: Yup i'll remove, think that was left over locally when tinkering with actions initally.

File: .github/workflows/deploy-to-pages.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # build step runs npm ci
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
+    steps:
+      - name: Use Node.js ${{ matrix.node-version }}
```
Comment (jdjkelly) [inline .github/workflows/deploy-to-pages.yml:26]:
```suggestion
      - uses: actions/checkout@v2
      - name: Use Node.js ${{ matrix.node-version }}
```

**Review by jdjkelly — APPROVED** [review]
Ok, I like it. I'll let you squash + merge. 

In the 'merged commit' you should just replace the written comment with something descriptive about this in a sentence or two

**Pull request merged**